fastlane/Fastfile

```diff
@@ -418,19 +418,12 @@ platform :ios do
     en_lproj = 'WordPress/Resources/en.lproj'
     ios_generate_strings_file_from_code(
       paths: ['WordPress/', 'Pods/WordPress*/', 'Pods/WPMediaPicker/', 'WordPressShared/WordPressShared/', 'Pods/Gutenberg/'],
-      exclude: ['*Vendor*', 'WordPress/WordPressTest/I18n.swift', 'WordPress/WordPressStatsWidgets/Views/Localization/AppLocalizedString.swift'],
+      exclude: ['*Vendor*', 'WordPress/WordPressTest/I18n.swift', 'WordPress/WordPressStatsWidgets/Views/Localization/AppLocalizedString.swift', 'WordPress/Classes/Utility/AppLocalizedString.swift'],
       routines: ['AppLocalizedString'],
       output_dir: en_lproj
     )
 
-    # @TODO: Remove once we migrated the codebase of the widgets to use the app bundle for .strings
```

Comment:
@AliSoftware – I needed to remove these in order to complete the code freeze – I believe these were resolve as part of #17630 and #17636 , but LMK if anything doesn't look right?

Reply:
That's correct and I should have removed those as part of those PRs, thanks for taking care of this.

There might still be an issue waiting to happen during `new_beta_release` and `finalize_release` when those will call the scripts downloading the translations from GlotPress and those scripts (which are still yet to be migrated) might still be trying to re-dispatch the keys to those widget targets based on those (now deleted) `Base.lproj` keys. Until that side of the migration of our scripts is over we might need to patch the extract-framework-strings scripts to stop doing that for those targets until I finalize the related work.